docker-compose.yml

```diff
@@ -33,19 +38,39 @@ services:
         interval: 10s
         timeout: 10s
         retries: 10
+
+# Runs the standard cartography image available at ghcr.io.
   cartography:
-    # As seen in docs, we build with `cd /path/to/cartography && docker build -t lyft/cartography .`
-    # and then run with `docker-compose up -d`.
-    image: lyft/cartography
+    image: ghcr.io/lyft/cartography:latest
     # EXAMPLE: Our ENTRYPOINT is cartography, running specific command to sync AWS
     # command: ["-v", "--neo4j-uri=bolt://neo4j:7687", "--aws-sync-all-profiles"]
-    user: cartography
     init: true
     restart: on-failure
     depends_on:
       - neo4j
     volumes:
-      - ~/.aws:/cartography/.aws/
+    # Provide AWS creds to the container
+      - ~/.aws:/var/cartography/.aws/
+    environment:
+      # Point to the neo4j service defined in this docker-compose file.
+      - NEO4J_URL=bolt://cartography-neo4j-1:7687
+
+# Intended to run local automated tests, custom sync scripts, and local changes.
+  cartography-dev:
+    # See dev instructions: we assume that you have built this with
+    # `docker build -t lyft/cartography-dev . -f dev.Dockerfile`.
+    # Do not push this image remotely!
+    image: lyft/cartography-dev
```

Comment:
Not a blocker.
You can add [build](https://docs.docker.com/reference/compose-file/build/#illustrative-example) to automatically build a new image (if code changed) during start-up.